agent/api/task/task_linux.go

```diff
@@ -76,14 +76,26 @@ func (task *Task) initializeCgroupResourceSpec(cgroupPath string, cGroupCPUPerio
 }
 
 // BuildCgroupRoot helps build the task cgroup prefix
-// Example: /ecs/task-id
+// Example v1: /ecs/task-id
+// Example v2: ECSTask$TASKID.slice
```

Comment:
Discussed offline, will change this to `ECSTasks-$TASKID.slice`. Using systemd parent-name.slice notation, this will make a parent slice called ECSTasks.slice for all ecs tasks with mem/cpu limits.